apps/dashboard/src/components/primitives/control-input/variable-plugin/variable-pill-widget.ts

```diff
@@ -108,6 +120,7 @@ export class VariablePillWidget extends WidgetType {
     const span = document.createElement('span');
     const content = document.createElement('span');
     content.textContent = this.variableName;
+    content.title = this.variableName;
```

Comment:
added the title to the variable name and filters for the case when text is with ellipsis